simul.spec

```
@@ -46,7 +49,7 @@ and coherence of parallel filesystems.
 %package mpich
 Summary: Simul for MPICH
 %if "%{_host_vendor}" == "suse"
-BuildRequires: mpich-devel%{?_isa} lua-lmod
+BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel glibc
```

You should never have to `BR: glibc`.  It's a base component that no Linux system can function without.

```suggestion
BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel
```